### DISCUSSION TITLE
Unable to change background image or image src in grapesjs

### DISCUSSION THREAD

**Discussion by prabhakaran8737** [GrapesJS/grapesjs#4409] [Q&A]
I have a open question in [SO](https://stackoverflow.com/questions/72747152/unable-to-change-background-image-or-image-src-in-grapesjs)

Here I have attached my [Jsfiddle](https://jsfiddle.net/prabhakaran8737/fwr3xapq/5/)

I am trying to change the background image in the header and also .

To do so I am configured [grapesjs-style-bg](https://github.com/artf/grapesjs-style-bg) still I am unable to do any changes related to the image. Am I missing out on something?

For <img..> I can understand it is because of this structure
```

<a data-gjs-editable="false" class=portfolio-item href=#>
    <span data-gjs-editable="false" class=caption>
        <span class=caption-content>
            <h2>Stationary</h2>
            <p class=mb-0>A yellow pencil with envelopes on a clean, blue backdrop!</p>
        </span>
    </span>
    <img class=img-fluid src=https://d3cm4d6rq8ed33.cloudfront.net/static/fprgam1277_hamara/orig/img/portfolio-1.jpg alt="">
</a>
```

Where the span is **positioned absolute** and so the image was just behind the span, so unable to edit it. I tried adding data-gjs-editable="false" as well still couldn't detect image.

**Comment by ronaldohoch** [comment]
Tryed some things but can't figure out why isn't working :/
Also, when i duplicate some other structure of your sample, it add the background :|
